src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -1,74 +1,57 @@
-import { AnyJson } from '@salesforce/ts-types';
-import child_process = require('child_process');
+import {Rule} from '../../types';
+import {AnyJson} from '@salesforce/ts-types';
+import {PmdSupport, PMD_LIB, PMD_VERSION} from './PmdSupport';
 import fs = require('fs');
 
-// TODO: Make these OS-agnostic and dynamic.
-const PMD_LIB = './dist/pmd/lib';
-const PMD_VERSION = '6.20.0';
+const PMD_CATALOGER_LIB = './dist/pmd-cataloger/lib';
```

Comment:
path.join()

Reply:
As I mentioned earlier, this should not be necessary.  Either way, the path constants should be standard here.  They should be split and rejoined later if needed for a platform-specific reason.